### PR TITLE
Include artifact building scripts

### PR DESCRIPTION
**Pull request by xrajesh**
## Description
1. `./hack/publish_build_from_tag.sh --publish`  script to build bundle and catalog images based on previous operand tag and operator tag.   @bparees Used for internal-preview images.
2. `./hack/rebuild_bundle_catalog.sh` developer script to rebuild the bundle folder and build bundle and catalog images from source.
3. Makefile to have update-bundle-catalog to update the artifacts for bundle and catalog builds

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: hack/update_bundle_catalog.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Build the bundle image
+echo "Updating bundle artifcts for image ${OPERATOR_IMAGE}"
+make bundle VERSION="${BUNDLE_TAG}" IMG="${OPERATOR_IMAGE}"
+
+echo "Adding bundle image to FBC using image ${BUNDLE_IMAGE}" 
+if [ -f "${CATALOG_FILE}" ]; then
+  rm -f "${CATALOG_FILE}"
+fi
+
+cat << EOF >> "${CATALOG_FILE}"
```
Comment (raptorsun) [inline hack/update_bundle_catalog.sh:26]: `cat << EOF > "${CATALOG_FILE}"` is shorter :)

File: hack/publish_build_from_tag.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Helper tool to build bundle and catalog image and push it to the quay.io registry
+# for given tag of the operator and operands.
```
Comment (bparees) [inline hack/publish_build_from_tag.sh:2]: as a followup i think you should make the tag an argument (and also allow use of a SHA instead of a tag)

**Pull request merged**